src/components/core/a11y/focus.ts

```diff
@@ -79,7 +91,7 @@ export class FocusTrap {
         const elementChildren: HTMLElement[] = Array.from(
           element.shadowRoot.children,
         ) as HTMLElement[];
-        const focusableElements = getFocusableElements(elementChildren, filterFunc);
+        const focusableElements = getFocusableElements(elementChildren, { filterFunc: filterFunc });
```

Comment:
nit pick, could probably be shortened
```suggestion
        const focusableElements = getFocusableElements(elementChildren, { filterFunc });
```